spaghetto/src/Spago/Purs.purs

```diff
@@ -42,16 +42,13 @@ getPurs =
   where
   dropStuff pattern = fromMaybe "" <<< Array.head <<< String.split (String.Pattern pattern)
 
-compile :: forall a. Set FilePath -> Array String -> Spago (PursEnv a) Unit
+compile :: forall a. Set FilePath -> Array String -> Spago (PursEnv a) (Either Cmd.ExecError Cmd.ExecResult)
 compile globs pursArgs = do
   { purs } <- ask
   let args = [ "compile" ] <> pursArgs <> Set.toUnfoldable globs
   logDebug [ "Running command:", "purs " <> String.joinWith " " args ]
-  Cmd.exec purs.cmd args Cmd.defaultExecOptions >>= case _ of
-    Right _r -> logSuccess "Build succeeded."
-    Left err -> do
-      logDebug $ show err
-      die [ "Failed to build." ]
+  Cmd.exec purs.cmd args $ Cmd.defaultExecOptions
+    { pipeStdout = false }
```

Comment:
`pipeStdout = false` is needed to prevent the JSON errors from `purs` being outputted to the console.

Reply:
I would expect this to take the output pipe and do things with the stream of the output so that things happen in real time rather than being buffered.
Is this how the current psa works?

Reply:
Just to clarify, when `purs` outputs its errors, it does so to `stdout`. When it outputs "Compiling ..." messages, it does so to `stderr`. So, the above code is saying:
1. pipe `stderr` to the parent's `stderr`, so that we see "Compiling ..." messages real-time.
2. don't pipe `stdout` to the parent's `stdout`, so that we don't see `{"warnings":[],"errors":[]}` in the console.

Are you not seeing a real-time output of the "compiling" messages?